doc/src/overrides.md

```diff
@@ -85,14 +88,19 @@ targets = [ "wasm32-unknown-unknown", "thumbv2-none-eabi" ]
 profile = "minimal"
 ```
 
-If the TOML format is used, the `[toolchain]` section is mandatory, and at
-least one property must be specified.
+The `[toolchain]` section is mandatory, and at least one property must be
+specified.
+
+For backwards compatibility, `rust-toolchain` files also support a legacy
+format that only contains a toolchain name without any TOML encoding, e.g.
+just `nightly-2021-01-21`. This format is not available in
+`rust-toolchain.toml` files.
 
-The `rust-toolchain` file is suitable to check in to source control. This file
-has to be encoded in US-ASCII (if you are on Windows, check the encoding and
-that it does not starts with a BOM).
+The `rust-toolchain.toml`/`rust-toolchain` files are suitable to check in to
+source control. The files have to be encoded in US-ASCII (if you are on
+Windows, check the encoding and that it does not starts with a BOM).
```

Comment:
Only `rust-toolchain` needs to be US-ASCII in theory `rust-toolchain.toml` should always be `utf-8`

Reply:
Do you mean only the legacy format of the `rust-toolchain` file or also when the TOML format is used? I assume it only affects the legacy format (see the latest commit), but please let me know if I'm wrong.

I'll rebase as soon as this is ready.

Reply:
Yep, that's correct.  Rebase and I think we're done.